tools/rpm2img

```diff
@@ -152,10 +152,17 @@ mv "${ROOT_MOUNT}/boot"/* "${BOOT_MOUNT}"
 
 # Set the Bottlerocket variant, version, and build-id
 SYS_ROOT="${ARCH}-bottlerocket-linux-gnu/sys-root"
-echo "PRETTY_NAME=\"${PRETTY_NAME} ${VERSION_ID}\"" >> "${ROOT_MOUNT}/${SYS_ROOT}/usr/lib/os-release"
-echo "VARIANT_ID=${VARIANT}" >> "${ROOT_MOUNT}/${SYS_ROOT}/usr/lib/os-release"
-echo "VERSION_ID=${VERSION_ID}" >> "${ROOT_MOUNT}/${SYS_ROOT}/usr/lib/os-release"
-echo "BUILD_ID=${BUILD_ID}" >> "${ROOT_MOUNT}/${SYS_ROOT}/usr/lib/os-release"
+VERSION="${VERSION_ID} (${VARIANT})"
+cat <<EOF >> "${ROOT_MOUNT}/${SYS_ROOT}/usr/lib/os-release"
+VERSION="${VERSION}"
+PRETTY_NAME="${PRETTY_NAME} ${VERSION}"
+VARIANT_ID=${VARIANT}
+VERSION_ID=${VERSION_ID}
```

Comment:
nit: should we also add `VARIANT`? Might be good to keep related fields together too.
```suggestion
PRETTY_NAME="${PRETTY_NAME} ${VERSION}"
VARIANT="${VARIANT}"
VARIANT_ID="${VARIANT}"
VERSION="${VERSION}"
VERSION_ID=${VERSION_ID}
```

Reply:
I'd hesitate to add VARIANT because it blurs the lines between a field and its corresponding _ID field, where _ID is the "technical" name... if we come up with "friendly" versions of variant names, I think that's what would go into VARIANT, but we don't have those now...

Reply:
I've typically seen VERSION close to the top of the os-release file (perhaps to make it less buried?)

Regarding VARIANT, I think I agree with @tjkirch that it would make more sense if it were a friendly name. That said, there is no reason we can't draft some up while we're at it.

Reply:
It feels like we use our variant identifiers interchangeably with names, and I'm not sure how they could be made more friendly.

I don't feel strongly about it, and I'm willing to hold off on `VARIANT` to leave the door open.